src/classes/query.py

```diff
@@ -257,6 +302,48 @@ def relative_path(self):
         # Convert path to relative (based on current working directory of Python)
         return os.path.relpath(file_path, info.CWD)
 
+    def getAudioData(self):
+        # Ensure that UI attribute exists
+        if "ui" not in self.data:
+            self.data["ui"] = {}
+            self.save()
+
+        audio_data = self.data["ui"].get("audio_data", False)
+        if audio_data and len(audio_data) > 1:
+            log.info("Audio Data already retrieved.")
+            return
+        if not audio_data:
+            # Placeholder value. Communicates that data is being retrieved
+            self.data["ui"]["audio_data"] = [-999]
+
+        # Do the audio data stuff.
+        temp_clip = openshot.Clip(self.data["path"])
+        temp_clip.Open()
+        temp_clip.Reader().info.has_video = False
+
+        sample_rate = temp_clip.Reader().info.sample_rate
+        samples_per_second = 20
+        sample_divisor = round(sample_rate / samples_per_second)
+
+        audio_data = []
+        for frame_num in range(1, temp_clip.Reader().info.video_length):
+            print(f"FileShow: frame {frame_num}")
```

Comment:
Remove print - or make it a `log.debug`

Reply:
Removed. I don't think logging every frame is useful information.